.github/workflows/dokken-unit-tests.yml

```diff
@@ -122,7 +122,7 @@ jobs:
           - ephemeral-drives-mounted
           - ephemeral-drives-skipped
           - sudo-configured
-          - /chrony-configured/
```

Comment:
Question, non-blocking: Just for the sake of curiosity, how this worked before this PR, I don't see any test respecting `chrony-configured` regex

Reply:
I guess it didn't...